api/v1alpha1/openapi.yaml

```diff
@@ -892,6 +892,38 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /api/v1/devices/{name}/console:
+    get:
```

Comment:
Do you think this is a GET, as in "get a console connection", or POST, as in "create a console connection".  The main reason I lean to POST is because we're updating the DB, which we usually don't with GET.  I don't have strong feelings about it though.  However, if we add DELETE to stop the connection, I would definitely move to POST.

Reply:
May be a post, we are creating something, but it's a bit weird, I don't think the user should actively delete connections, they should naturally go away once the session has been used (client connected and then disconnected, or remote process exited), or after timeout, or we will end with lots of dangling connections.

Reply:
Yes, the connection should be deleted when the session ends / times out, but consider the scenario where an admin would like to force-disconnect sessions.

Reply:
I will leave it as is now, since we want to remove the method altogether in the final solution (see comments in https://github.com/flightctl/flightctl/pull/313/files#r1674031310 )

I will make a note about admin closing console connections.